TACC/Core-CMS-Resources
# GH-70: Rename "Fullwidth" to "Full Width"

# Overview / Issues

- Closes #70

# Changes

See https://github.com/TACC/Core-CMS-Resources/pull/71/commits.

# Testing

1. Recreate https://pprd.protx.tacc.utexas.edu/ locally.
2. Update local secrets: rename "Fullwidth" to "Full Width" in `_CMS_TEMPLATES`.
2. Ensure local instance of ProTX homepage has "Full Width" template.
3. Ensure local instance of ProTX homepage still loads the same ProTX custom assets as [pre-prod](https://pprd.protx.tacc.utexas.edu/?edit).\*†

> \* One known discrepancy is `site.fonts.css` which is an asset that did not exist in `main` branch since last deploy of ProTX.
> † The style differences are caused by [`main` branch's new typography](https://github.com/TACC/Core-CMS/pull/312) since last deploy of ProTX.

# Screenshots

| Site | Template | Change | Screenshot | Notes |
| --- | --- | --- | --- | --- |
| __ProTX__ | Full Width | only rename | ![GH-70 ProTX Fullwidth - CSS Load](https://user-images.githubusercontent.com/62723358/131407875-b57e0ddc-ca90-45e6-90f3-7ad5637697c6.png) ![GH-70 ProTX Fullwidth - Template](https://user-images.githubusercontent.com/62723358/131407881-df0acb76-1004-42b2-9278-3c2b3ebd86bb.png) | rename pending [FP-1189](https://jira.tacc.utexas.edu/browse/FP-1189)
| __ProTX__ | Standard | new | N/A | no immediate usage planned |
| __A2CPS__ | Full Width | only rename | … | rename pending [FP-1189](https://jira.tacc.utexas.edu/browse/FP-1189)
| __A2CPS__ | Standard | new | N/A | no immediate usage planned |

# Notes

## ProTX

- Even though the custom stylesheets are empty now, I believe they will nto be empty in the future. I will likely need to add the header background color change to ProTX custom CSS, because https://github.com/TACC/Core-CMS/issues/191 will not be finished soon.
